etcdctl/ctlv3/command/check.go

```diff
@@ -126,7 +126,7 @@ func NewCheckPerfCommand() *cobra.Command {
 	}
 
 	// TODO: support customized configuration
-	cmd.Flags().StringVar(&checkPerfLoad, "load", "s", "The performance check's workload model. Accepted workloads: s(small), m(medium), l(large), xl(xLarge)")
+	cmd.Flags().StringVar(&checkPerfLoad, "load", "s", "The performance check's workload model. Accepted workloads: s(small), m(medium), l(large), xl(xLarge). Different workload models use different configurations in terms of number of clients and expected throughtput.")
```

Comment:
It indeed means "expected throughput" to me. We set an expected throughput, then measure the real throughput.

For example, if [the actual throughput is less than 90% of the expected throughput](https://github.com/etcd-io/etcd/blob/fc690531e213d916e666d00fc072a30fbac86a0c/etcdctl/ctlv3/command/check.go#L242), then the test fails.

Reply:
Probably it makes more sense to clearly document the standard on the pass of test. 
1. throughput >= 90%
2. s.Slowest > 0.5 
3. s.Stddev > 0.1

[check.go#L242-L254](https://github.com/etcd-io/etcd/blob/fc690531e213d916e666d00fc072a30fbac86a0c/etcdctl/ctlv3/command/check.go#L242-L254)

Reply:
I like the document as such as a concise doc. Adding more details in the README is also a good idea. Also, another thought is to add more doc in the command [function](https://github.com/etcd-io/etcd/blob/fc690531e213d916e666d00fc072a30fbac86a0c/etcdctl/ctlv3/command/check.go#L140) to clarify how it works - and then just point to that in the readme. In case, in future, we add new threshold or change in the existing values then we don't have to update doc. Thanks @patrocinio

Reply:
Thanks  @spzala for the feedback.

> add more doc in the command [function](https://github.com/etcd-io/etcd/blob/fc690531e213d916e666d00fc072a30fbac86a0c/etcdctl/ctlv3/command/check.go#L140) to clarify how it works - and then just point to that in the readme

This might not work. When we update the source code, the link might be out of date. So the best approach would be to get all detailed included in the README directly instead of just adding a link.

Reply:
@ahrtr thanks and yes I agree with you! It's more about pointing to the function, not a particular LOC. ReadMe sounds good to me.